x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/get_baseline_and_deviation_rates.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { LOG_RATE_ANALYSIS_TYPE } from '@kbn/aiops-log-rate-analysis';
+
+export function getLogRateChange(
+  analysisType: typeof LOG_RATE_ANALYSIS_TYPE[keyof typeof LOG_RATE_ANALYSIS_TYPE],
+  baselineBucketRate: number,
+  deviationBucketRate: number
+) {
+  const logRateChange =
+    baselineBucketRate > 0
+      ? analysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE
+        ? `${Math.round((deviationBucketRate / baselineBucketRate) * 100) / 100}x higher`
+        : `${Math.round((baselineBucketRate / deviationBucketRate) * 100) / 100}x lower`
```

Comment:
Guess we also need to add i18n to those strings.

Reply:
Can you use https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/common/util/metric_change_description.ts for these, as used for the anomalies table? That will use 'More than 100x higher' if the factor is `> 100`, which is probably appropriate to use here too? Or if you can't call that same code, then use the same rules around precision so that e.g. `x14.23 higher` becomes `x14 higher`.

Reply:
Updated to round and to add localization in ff861eb3f280d3812b7536fcbabcd06eced039cc